### PR TITLE
Created builder package with PointBuilder,RouteBuilder,UserBuilder classes

### PR DESCRIPTION
**Pull request by AikaKalil**
Created builder package with PointBuilder,RouteBuilder,UserBuilder classes

### CODE REVIEW

File: src/main/java/org/example/model/builder/UserBuilder.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
+package org.example.model.builder;
+import org.example.model.Observer.DesiredPath;
+import org.example.model.User;
+public class UserBuilder {
+        private Integer id;
```
Comment (remynewton) [inline src/main/java/org/example/model/builder/UserBuilder.java:5]: You need to fix the formatting. Just go to the file and type command + option + L if you're using the Mac OS and Ctrl + Alt + L if you're using the either Windows or Linux OS.

**Review by remynewton — APPROVED** [review]
LGTM!

**Review by kabuki-mono — APPROVED** [review]
LGTM 👍, please note James' suggestion and change the title of your pull request to a general description instead of your name

**Review by TomashevskiiVladimirSolvd — APPROVED** [review]
LGTM.

**Pull request merged**